### PR TITLE
css: add note about retrieving styles for detached elements

### LINKED ISSUE #653: Add note to .css() about retrieving styles for detached elements
Context: https://github.com/jquery/jquery/issues/2086

There is no useful style information before a node is connected to the DOM. jQuery takes a queue here from the native behavior and allows an error to be thrown.

### PR DESCRIPTION
**Pull request by arthurvr**
Would fix #653

### CODE REVIEW

**Comment by arthurvr** [comment]
@timmywil Would you mind reviewing this?

**Comment by arthurvr** [comment]
Somebody who wants to give this a quick review?

File: entries/css.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@
       <p>Also, jQuery can equally interpret the CSS and DOM formatting of multiple-word properties. For example, jQuery understands and returns the correct value for both <code>.css( "background-color" )</code> and <code>.css( "backgroundColor" )</code>.</p>
 	  <p>Note that the <em>computed style</em> of an element may not be the same as the value specified for that element in a style sheet. For example, computed styles of dimensions are almost always pixels, but they can be specified as em, ex, px or % in a style sheet. Different browsers may return CSS color values that are logically but not textually equal, e.g., #FFF, #ffffff, and rgb(255,255,255).</p>
       <p>Retrieval of shorthand CSS properties (e.g., <code>margin</code>, <code>background</code>, <code>border</code>), although functional with some browsers, is not guaranteed. For example, if you want to retrieve the rendered <code>border-width</code>, use: <code>$( elem ).css( "borderTopWidth" )</code>, <code>$( elem ).css( "borderBottomWidth" )</code>, and so on.</p>
+      <p>An element should be connected to the DOM when calling <code>.css()</code> on it. If it isn't, jQuery will allow an error to be thrown.</p>
```
Comment (AurelioDeRosa) [inline entries/css.xml]: I'd replace "will allow an error to be thrown" with "will thrown an error". It seems more readable.

Comment (dmethvin) [inline entries/css.xml]: Perhaps "may throw an error" instead? That says it might but doesn't guarantee it will. :smile:

**Comment by arthurvr** [comment]
@dmethvin I pushed the update. This LGTY?

**Comment by dmethvin** [comment]
Yep, thanks!

**Pull request closed without merging**